src/views/Correction.vue

```diff
@@ -692,14 +695,14 @@ export default class Correction extends Mixins(
         correctedFilingId: this.correctedFilingId,
         correctedFilingType: this.origFiling.header.name,
         correctedFilingDate: this.dateToYyyyMmDd(this.apiToDate(this.origFiling.header.date)),
-        comment: `${this.defaultComment}\n${this.detailComment}`
+        comment: `${this.defaultComment}\n${this.detailComment}`,
+        type: this.correctionType
```

Comment:
I believe you don't need to change this page since it's not impacted by the correction type at this time. (Or does the API require this new property? And if it does then it shouldn't be optional in src/interfaces/correction-filing-interface.ts.)

Looks like this variable is null anyway since it's not set anywhere in this page.

Reply:
You are right. This variable is not used. It's now fixed.

Reply:
There are still a couple of diffs in this file. Could you revert them please?